include/dxc/Support/dxcapi.use.h

```diff
@@ -19,9 +19,51 @@ namespace dxc {
 extern const char *kDxCompilerLib;
 extern const char *kDxilLib;
 
-// Helper class to dynamically load the dxcompiler or a compatible libraries.
-class DxcDllSupport {
+// Interface for common dll operations
+class DllLoader {
+public:
```

Comment:
What's the thinking between the multiple `public`, `protected` and `public` sections here?  There's some pure virtual functions defined in the first public, but there's also some in the second.

Reply:
Merged into one.